### PR TITLE
GPS docs update

### PR DESCRIPTION
**Pull request by hamishwillee**
- All GPS/compass docs now listed in one big table. This has columns for the main features of interest - gps module, compass module, RTK support, yaw support etc.
- Other docs link to this table. I think will be easier to maintain.
- MOved "devices that support yaw" out of RTK into the main doc.

I _think_ this is a better overall structure.

### CODE REVIEW

File: en/advanced/rtk_gps.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The datalink should typically be able to handle an uplink rate of 300 bytes per
 
 ## Supported RTK GPS modules
 
-PX4 currently only supports the single-frequency (L1) u-blox M8P based GNSS receivers for RTK.
+PX4 currently only supports the single-frequency (L1) u-blox M8P based GNSS receivers for RTK. <!-- F9P? -->
```
Comment (hamishwillee) [inline en/advanced/rtk_gps.md:24]: @bkueng I do not believe this section (in the developer docs) is true. We support F9P too now! Can you provide a new comment about our level of support - ie what modules, what frequencies?

Also a quick scan of the rest of this doc for anything else out of date? Feel free to push in any changes to the branch.

File: en/gps_compass/README.md [hunk 2]
```diff
@@ -22,55 +17,66 @@ The internal compass *may* be useful on larger vehicles (e.g. VTOL) where it is
...
+[Avionics Anonymous UAVCAN GNSS/Mag](https://www.tindie.com/products/avionicsanonymous/uavcan-gps-magnetometer/) | SAM-M8Q | MMC5983MA | | |
+3DR u-blox GPS with Compass kit (discontinued) | LEA-6H | &check; | | | 
+[CUAV C-RTK GPS](../gps_compass/rtk_gps_cuav_c-rtk.md) | M8P/M8N | &check; | &check; | |
+[Drotek XL RTK GPS](../gps_compass/rtk_gps_drotek_xl.md) | M8U | LIS3MDL | &check; | | 
+[Femtones MINI2 Receiver](../gps_compass/rtk_gps_fem_mini2.md) | FB672, FB6A0 | &check; | &check; | | 
+[Freefly RTK GPS](../gps_compass/rtk_gps_freefly.md) | F9P | IST8310 | &check; | 
+[Here+ RTK GPS](../gps_compass/rtk_gps_hex_hereplus.md) | M8P | HMC5983 | &check; | | 
+[Holybro H-RTK F9P GNSS](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md) | F9P | IST8310 | &check; | | 
+[Holybro H-RTK M8P GNSS](../gps_compass/rtk_gps_holybro_h-rtk-m8p.md) | M8P | IST8310 | &check; | | 
+[SparkFun GPS-RTK2 Board - ZED-F9P](https://www.sparkfun.com/products/15136) | F9P |  ? | &check; | | &check;
```
Comment (hamishwillee) [inline en/gps_compass/README.md:44]: @bkueng Does this board have a magnetometer?

File: en/gps_compass/README.md [hunk 3]
```diff
@@ -22,55 +17,66 @@ The internal compass *may* be useful on larger vehicles (e.g. VTOL) where it is
...
+[Drotek SIRIUS RTK GNSS ROVER (F9P)](https://store-drotek.com/911-1010-sirius-rtk-gnss-rover-f9p.html#/158-sensor-no_magnetometer) | F9P | RM3100 | &check; | | &check;
+[mRo u-blox ZED-F9 RTK L1/L2 GPS](https://store.mrobotics.io/product-p/mr-m10020-a.htm) | F9P | &cross; | &check; | | &check;
+[Trimble MB-Two](../gps_compass/rtk_gps_trimble_mb_two.md) | F9P | ? | &check; | &check; | |
```
Comment (hamishwillee) [inline en/gps_compass/README.md:47]: I assume the trimble does NOT have a magnetometer?

File: en/gps_compass/README.md [hunk 4]
```diff
@@ -22,55 +17,66 @@ The internal compass *may* be useful on larger vehicles (e.g. VTOL) where it is
 On small vehicles an external compass is almost always required.
 :::
 
-## Combined GPS/Compass Options
+## Supported GPS and/or Compass
+
+Device | GPS | Compass | [RTK](../gps_compass/rtk_gps.md) | [Yaw Output](#configuring-gps-as-yaw-heading-source) | [Dual FP9 GPS Heading](../gps_compass/u-blox_f9p_heading.md)
+:--- | :---: | :---:  | :---:  | :---:  | :---: | :---:
+[Hobbyking u-blox Neo-M8N GPS with Compass](https://hobbyking.com/en_us/ublox-neo-m8n-gps-with-compass.html?gclid=Cj0KCQjwqM3VBRCwARIsAKcekb3ojv1ZhLz1-GuvCsUuGT8ZZuw8meMIV_I6pgUCj6DJRzHBY9OApekaAgI5EALw_wcB&gclsrc=aw.ds&___store=en_us) | Neo-M8N | &check; | | |
```
Comment (hamishwillee) [inline en/gps_compass/README.md:24]: Some refer to Neo-8MN, which I assume is the "proper name", while others refer to 8MN. OK if I use 8MN everywhere?

**Review by bkueng — COMMENTED** [review]
Thanks @hamishwillee, looks much better

File: en/advanced/rtk_gps.md (hunk 1 above)
Comment (hamishwillee) [reply en/advanced/rtk_gps.md]: Thank you!

**Comment by hamishwillee** [comment]
Thanks @bkueng - I think a bit better too - certainly "more extensible". Merged.

**Pull request merged**